dist/OutSystemsUI.js

```diff
@@ -19131,11 +19131,15 @@ var Providers;
                             if (((_a = this.StartingSelection) === null || _a === void 0 ? void 0 : _a.length) > 0) {
                                 if (this.AllowMultipleSelection) {
                                     for (const option of this.StartingSelection) {
-                                        selectedKeyvalues.push(option.value);
+                                        if (option.value !== OSFramework.OSUI.Constants.EmptyString) {
+                                            selectedKeyvalues.push(option.value);
+                                        }
                                     }
                                 }
                                 else {
-                                    selectedKeyvalues.push(this.StartingSelection[0].value);
+                                    if (this.StartingSelection[0].value !== '') {
```

Comment:
Shouldn't we use the OSFramework.OSUI.Constants.EmptyString here too?

Reply:
done!